src/appleseed/renderer/modeling/project/projectfilewriter.cpp

```diff
@@ -110,7 +110,6 @@ namespace renderer
 namespace
 {
     // Floating-point formatting settings.
-    const char* VectorFormat     = "%.15f";
```

Comment:
Surprised about this one. How do we control precision when writing vectors then?